mitmproxy/addons/tutorialapp/templates/mitmproxy/index.html

```diff
@@ -0,0 +1,210 @@
+            Let's assume the following scenario: We have a cat called Lucy.
+            We registered Lucy for the famous global <em>Shocked Animal Challenge</em>.
+            The organizers use a custom web application that allows people to vote for the participants.
+            This voting application is embedded in the right sidebar of this page.
+            Let's see if we can use mitmproxy to get Lucy some extra votes.
+            <!-- Obviously we want Lucy to win. -->
+
+            <!-- One of our clients is the host of the global <em>Shocked Animal challenge</em>.
+            We are the organizers of the global Shocked Animal challenge.
+            Thus, we have created a fictitious web application -->
+        </p>
+
+        <p>
+            <a class="btn btn-sm btn-outline-secondary" data-toggle="collapse" href="#collapse-how-tutorial-app-implemented" role="button" aria-expanded="false" aria-controls="collapse-how-tutorial-app-implemented">
+              Optional: How is this tutorial implemented?
+            </a>
+        </p>
+
+        <div class="collapse mb-4" id="collapse-how-tutorial-app-implemented">
+            <div class="card card-body">
+                <p class="mb-0">
+                    This tutorial is implemented as a mitmproxy <a href="https://docs.mitmproxy.org/stable/addons-overview/">addon</a>.
+                    Addons allow to easily extend the functionality of mitmproxy using its API.
+                    The tutorial addon intercepts all HTTP requests.
+                    If the request's HTTP <code>Host</code> header is <code>tutorial.mitm.it</code>, the request is redirected to the tutorial's web application.
+                    The web application itself is implemented using <a href="https://palletsprojects.com/p/flask/">Flask</a>.
+                    All other intercepted requests are forwarded to their original destination without modification.
+                    The core of the tutorial addon has less than 100 lines of code.
+                    The details go beyond the scope of this tutorial, but it should give you a glance of the advanced features you can implement with mitmproxy.
+                </p>
+            </div>
+        </div>
+
+        <!-- <p>
+            Let's see if mitmproxy can help us make sure that Lucy wins.
+            We first need to understand how the application works.
+            So let's start with investigating what happens when we submit a vote.
+        </p> -->
+
+        <h2 id="view-flows" class="pt-3 pb-3 mb-3 border-bottom">mitmproxy's User Interface</h2>
+
+        <!-- <p>We suggest that you position the windows next to ...</p> -->
+
+        <p>
+            First of all,we needs to get familiar with mitmproxy's user interface.
+            Open the terminal window in which you started mitmproxy.
+            You should see the <span class="badge badge-info">Default View</span> of mitmproxy, which shows a table-like list of flows.
```

Comment:
Nit: I think using a badge here doesn't really work out. Maybe just `text-primary`?

Reply:
I'll check what works best. I agree that a badge might emphasize it too much and might be misleading.